src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -491,11 +491,21 @@ namespace webnn_native { namespace ie {
         return {};
     }
 
+    std::string Graph::GetDeviceName() {
+        ml::DevicePreference devicePreference = mContextOptions.devicePreference;
+        if (devicePreference == ml::DevicePreference::Gpu) {
+            return "GPU";
+        } else {
+            return "CPU";
+        }
+    }
+
     void Graph::CompileImpl(BuildGraphCallbackDelegate delegate) {
         // TODO(junwei): We may leverage https://dawn-review.googlesource.com/c/dawn/+/36360 to
         // implement async compilation as standle-alone component.
         // Create compilation for IE backend.
-        IEStatusCode code = IE(ie_create_compilation)(mIeModel, &mIeCompilation);
+        IEStatusCode code =
```

Comment:
char * deviceName = devicePreference == ml::DevicePreference::Gpu || Default ? "GPU" : "CPU";

Reply:
For OpenVINO, I suppose the default should be `CPU`.

Reply:
Yes, I agree, I define in [GetDeviceName() function](https://github.com/webmachinelearning/webnn-native/pull/36/files/948008010ab1d6ab599bcfd6ab2044eb7641468e#diff-8c735e422d57ddc0234da2ed876a76e5bf398a447c94c4e819b14f9d8fd7c812R463). Thanks! @huningxin 

For this suggestion: `char * deviceName = devicePreference == ml::DevicePreference::Gpu || Default ? "GPU" : "CPU";` I reserve [GetDeviceName() function](https://github.com/webmachinelearning/webnn-native/pull/36/files/948008010ab1d6ab599bcfd6ab2044eb7641468e#diff-8c735e422d57ddc0234da2ed876a76e5bf398a447c94c4e819b14f9d8fd7c812R463) to get name and also to prepare for more device preferences/power preferences selection in the future. Thanks! @fujunwei Does it make sense?